gravity-forms/gwaft-template-collapsible.php

```diff
@@ -25,7 +25,14 @@
 	$pages = $data['form']['pagination']['pages'];
 	$page_groups = array();
 	foreach ( $data['items'] as $item ) {
-		$page_groups[ $item['field']['pageNumber'] ][] = $item;
+		$field = $item['field'];
+		// Skip hidden fields.
+		if ( $field->type == 'hidden' || $field->visibility == 'hidden' ) {
```

Comment:
I can't really see a situation in which type casting would be useful here 🧐 

```suggestion
		if ( $field->type === 'hidden' || $field->visibility === 'hidden' ) {
```